src/types/global.d.ts

```diff
@@ -16,4 +16,15 @@ declare module '*.svg' {
     export default content;
 }
 
+declare global {
```

Comment:
Could you change `src/libs/cropOrRotateImage/types.ts` and `src/libs/cropOrRotateImage/index.ts` to stop using `FileWithUri` and delete that type?